crates/uv-python/src/interpreter.rs

```diff
@@ -550,6 +551,61 @@ impl ExternallyManaged {
     }
 }
 
+#[derive(Debug, Error)]
+pub struct UnexpectedResponseError {
+    pub(super) err: serde_json::Error,
+    pub(super) stdout: String,
+    pub(super) stderr: String,
+    pub(super) path: PathBuf,
+}
+
+impl Display for UnexpectedResponseError {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        writeln!(
+            f,
+            "Querying Python at `{}` did not return the expected data\n{}",
+            self.path.display(),
+            self.err
+        )?;
+        let delim = "---";
+        if !self.stdout.trim().is_empty() {
+            writeln!(f, "{} stdout:\n{}", delim, self.stdout)?;
+        }
+        if !self.stderr.trim().is_empty() {
+            write!(f, "{0} stderr:\n{1}\n{0}", delim, self.stderr)?;
```

Comment:
Is `write` here vs. `writeln` above intentional?

Reply:
No, it wasn't fixed now. Thank you